.ci/operatorhub.sh

```diff
@@ -21,104 +29,45 @@ CRD_FILE=deploy/crds/jaegertracing.io_jaegers_crd.yaml
-* [x] Is operator [icon](https://github.com/operator-framework/community-operators/blob/master/docs/contributing.md#operator-icon) set?
-
-### Updates to existing Operators
-
-* [x] Did you create a ci.yaml file according to the [update instructions](https://github.com/operator-framework/community-operators/blob/master/docs/contributing.md#updating-your-existing-operator)?
-* [x] Is your new CSV pointing to the previous version with the replaces property if you chose replaces-mode via the updateGraph property in ci.yaml?
-* [x] Is your new CSV referenced in the [appropriate channel](https://github.com/operator-framework/community-operators/blob/master/docs/contributing.md#bundle-format) defined in the package.yaml or annotations.yaml ?
-* [ ] Have you tested an update to your Operator when deployed via OLM?
-* [x] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing.md#sign-your-work)?
-
-### Your submission should not
-
-* [x] Modify more than one operator
-* [x] Modify an Operator you don't own
-* [x] Rename an operator - please remove and add with a different name instead
-* [x] Submit operators to both upstream-community-operators and community-operators at once
-* [x] Modify any files outside the above mentioned folders
-* [x] Contain more than one commit. **Please squash your commits.**
-
-### Operator Description must contain (in order)
-
-1. [x] Description about the managed Application and where to find more information
-2. [x] Features and capabilities of your Operator and how to use it
-3. [x] Any manual steps about potential pre-requisites for using your Operator
-
-### Operator Metadata should contain
-
-* [x] Human readable name and 1-liner description about your Operator
-* [x] Valid [category name](https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md#categories)<sup>1</sup>
-* [x] One of the pre-defined [capability levels](https://github.com/operator-framework/operator-courier/blob/4d1a25d2c8d52f7de6297ec18d8afd6521236aa2/operatorcourier/validate.py#L556)<sup>2</sup>
-* [x] Links to the maintainer, source code and documentation
-* [x] Example templates for all Custom Resource Definitions intended to be used
-* [x] A quadratic logo
-
-Remember that you can preview your CSV [here](https://operatorhub.io/preview).
-
---
-
-<sup>1</sup> If you feel your Operator does not fit any of the pre-defined categories, file an issue against this repo and explain your need
-
-<sup>2</sup> For more information see [here](https://github.com/operator-framework/operator-sdk/blob/master/doc/images/operator-capability-level.svg)
-EOM
+    tmpfile=$(mktemp /tmp/Update-Jaeger-to-${VERSION}.XXX)
+    echo "Update Jaeger to v${VERSION}" > ${tmpfile}
+    curl https://raw.githubusercontent.com/${dest}/main/docs/pull_request_template.md >> "${tmpfile}"
```

Comment:
Sorry, what I meant in the previous comment is to use *our* template file. Our template file has items checked, like this:

```md
 * [x] A quadratic logo
```

I think we had a logic somewhere where we would check whether our template was still in sync with upstream, and fail the target if it isn't.

Reply:
Ok, I missunderstood your comment, 

Reintroduced the behaviour , now the check template is sending.